src/common/services/session/sessionModal.controller.spec.js

```diff
@@ -0,0 +1,62 @@
+import angular from 'angular';
+import 'angular-mocks';
+import module from './sessionModal.controller';
+
+describe( 'sessionModalController', function () {
+  beforeEach( angular.mock.module( module.name ) );
+  let $ctrl, uibModalInstance, state;
+
+  beforeEach( inject( function ( $controller, ) {
+    uibModalInstance = {
+      close:   angular.noop,
+      dismiss: angular.noop
+    };
+    state = 'sign-in';
+    $ctrl = $controller( module.name, {
+      $uibModalInstance: uibModalInstance,
+      state:             state
+    } );
+    spyOn( uibModalInstance, 'close' );
+    spyOn( uibModalInstance, 'dismiss' );
```

Comment:
This works fine but to not have to use the `angular.noop` above, you could use `createSpy` or `createSpyObj`:

```
uibModalInstance = {
    close:   jasmine.createSpy('close'),
    dismiss: jasmine.createSpy('dismiss')
};
```
or
```
uibModalInstance = jasmine.createSpyObj('uibModalInstance', ['close', 'dismiss']);
```
I think the first parameter in each jasmine function is just used in error reporting...